terraform/cloudsql/mysql/provision/outputs.tf

```diff
@@ -14,3 +14,7 @@ output "sslkey" {
   value     = google_sql_ssl_cert.client_cert.private_key
   sensitive = true
 }
+output "status" { value = format("%s database created URL: https://console.cloud.google.com/sql/instances/%[1]s/overview?project=%s",
```

Comment:
thought: I find the name of this field to be somewhat deceptive, as it doesn't actually reflect the status, but rather provides a (potentially broken) link to the dashboard.

Reply:
Thanks for your comment. We can talk about it on Monday and we'll make a decision